src/sql/workbench/contrib/scripting/browser/scripting.contribution.ts

```diff
@@ -287,6 +288,27 @@ MenuRegistry.appendMenuItem(MenuId.ObjectExplorerItemContext, {
 		TreeNodeContextKey.NodeType.isEqualTo(NodeType.TableValuedFunction))
 });
 
+MenuRegistry.appendMenuItem(MenuId.ObjectExplorerItemContext, {
+	group: 'inline',
+	order: 7,
+	command: {
+		id: commands.OE_REFRESH_COMMAND_ID,
+		title: localize('refreshNode', "Refresh"),
+		icon: Codicon.refresh
+	},
+	when: ContextKeyExpr.or(
```

Comment:
why only do refresh for these? I think Refresh should be available for all node types

Reply:
@alanrenmsft , I just copied the action above and changed the group.